pages/learn/deploy/deployment.md

```diff
@@ -49,13 +49,13 @@ __Note__: Refer to the [`{{$names.company.lower}} build`][cli-build-reference] a
 
 ### Overview
 
-The `git push {{$names.company.lower}} master` method of deployment is the original deployment mechanism for {{$names.cloud.lower}}. While we continue to support git push, it is considered a legacy method for pushing code to an application, and if possible you should use [{{$names.company.lower}} push](#balena-push) as it makes for a consistent workflow and methodology.
+The `git push {{$names.company.lower}} master` method of deployment is the original deployment mechanism for {{$names.cloud.lower}}. While we continue to support git push, it is considered a legacy method for pushing code to an fleet, and if possible you should use [{{$names.company.lower}} push](#balena-push) as it makes for a consistent workflow and methodology.
 
 The `git push` workflow requires that you have [git][git] installed on your development machine and that you have an SSH key [setup on your {{$names.cloud.lower}} account][add-ssh-key].
 
 ![how git push works](/img/common/deployment/git-push.png)
 
-Then, simply add your {{$names.cloud.lower}} app's git endpoint to your local git repository via `git remote add {{$names.company.lower}} <application git endpoint>` . You can find the application git remote endpoint at the top-right corner of the application page of the web dashboard.
+Then, simply add your {{$names.cloud.lower}} app's git endpoint to your local git repository via `git remote add {{$names.company.lower}} <fleet git endpoint>` . You can find the fleet git remote endpoint at the top-right corner in the releases tab of the dashboard.
```

Comment:
OK for now but this will be changing soon.

Reply:
Actually something like this would be future proof

```suggestion
Then, simply add your {{$names.cloud.lower}} app's git endpoint to your local git repository via `git remote add {{$names.company.lower}} <fleet git endpoint>` . You can find the fleet git remote endpoint by clicking the 'Create release' button in the releases tab of the dashboard.
```